.build/startup.sh

```diff
@@ -31,11 +31,19 @@ fi
 
 cd /opt/hue-emulator
 
+if [ $3 == "true" ]; then
```

Comment:
Shouldn't this be `[[ ... ]]`? Double square brackets? Or put the `$3` in double quotes. Currently, it is error-ing out at this line and always going into the else block. Error: `[: ==: unary operator expected`